transport/http/outbound.go

```diff
@@ -445,6 +400,118 @@ func getYARPCErrorFromResponse(response *http.Response, bothResponseError bool)
 	)
 }
 
+// RoundTrip implements the http.RoundTripper interface, making a YARPC HTTP outbound suitable as a
+// Transport when constructing an HTTP Client. An HTTP client is suitable only for relative paths to
+// a single outbound service. The HTTP outbound overrides the host:port portion of the URL of the
+// provided request.
+//
+// Sample usage:
+//
+//  client := http.Client{Transport: outbound}
+//
+// Thereafter use the Golang standard library HTTP to send requests with this client.
+//
+//  ctx := context.Background()
+//  ctx, cancel := context.WithTimeout(ctx, time.Second)
```

Comment:
nit: any advantage to not in-lining the `context.Background()` in the docs, since that's what most real-world use cases look like?

Reply:
Only subjective if we aren't looking at the data,

Reply:
I mean, value of consistency in this case is subjective ;)